bin/start-user-internet-available

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
```

Comment:
Given that this is a fresh new repo, would it make sense to switch to Python3 from the ground up?

Reply:
It would, but unfortunately I had to use some functions from kano-toolset that were written in python2